admin/templates/configuration/bin/qserv-restart.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+{{QSERV_RUN_DIR}}/bin/qserv-stop.sh
+{{QSERV_RUN_DIR}}/bin/qserv-start.sh
+{{QSERV_RUN_DIR}}/bin/qserv-status.sh
```

Comment:
It's very convenient to have "status" there also, to see the PID for attaching a debugger, tho...